### ISSUE TITLE
problem finding socket

### ISSUE THREAD

**Issue by sdahlbac** [DarthSim/overmind#107]
Suddenly overmind stopped working, (yesterday it worked flawlessly):

```
overmind start -f ../Procfile -l jwt_server
system               | Tmux socket name: overmind-redacted-u6cAOujqYy6J-80Y9BZMHW
system               | Tmux session ID: redacted
system               | Listening at ./.overmind.sock
error connecting to /private/tmp/tmux-501/overmind-redacted-u6cAOujqYy6J-80Y9BZMHW (No such file or directory)
```
```
ls -la /private/tmp/tmux-501/overmind-redacted-u6cAOujqYy6J-80Y9BZMHW
srw-rw----  1 sdahlbacka  wheel  0 Mar 19 12:59 /private/tmp/tmux-501/overmind-redacted-u6cAOujqYy6J-80Y9BZMHW
```

trying to run over tcp does not seem to work either

```
OVERMIND_SOCKET="0.0.0.0:4321" OVERMIND_NETWORK="tcp" overmind start -f ../Procfile -l jwt_server
system               | Tmux socket name: overmind-redacted-xVwLovQ8gO_n1mMzspwBFa
system               | Tmux session ID: redacted
system               | Listening at 0.0.0.0:4321
error connecting to /private/tmp/tmux-501/overmind-redacted-xVwLovQ8gO_n1mMzspwBFa (No such file or directory)
```

overmind --version
Overmind version 2.2.1

brew list tmux
/usr/local/Cellar/tmux/3.1c_1/bin/tmux
/usr/local/Cellar/tmux/3.1c_1/etc/bash_completion.d/tmux
/usr/local/Cellar/tmux/3.1c_1/share/man/man1/tmux.1
/usr/local/Cellar/tmux/3.1c_1/share/tmux/example_tmux.conf

osx big sur 11.2 20D64

**Comment by DarthSim** [comment]
Hey Simon!

It looks like a problem with tmux, and I bet it caused by something hidden by `redacted`. Try these two things:

- Run tmux manually using the same socket name: `tmux -L overmind-redacted-xVwLovQ8gO_n1mMzspwBFa` (replace `redacted` with the actual value). If I guessed right, this should fail.
- Run Overmind with a custom title: `OVERMIND_TITLE="lorem" overmind start -f ../Procfile -l jwt_server`. if I guessed right, this should work.

**Comment by sdahlbac** [comment]
- running `tmux -L  overmind-redacted-uEyVT5yZk1Nq6OaWGOylCo` opens up a tmux session
- running `OVERMIND_TITLE="lorem" overmind start -f ../Procfile -l jwt_server` 

results in 

```
system               | Tmux socket name: overmind-lorem-8gIald16MpjMK0sFPIg8Of
system               | Tmux session ID: lorem
system               | Listening at ./.overmind.sock
error connecting to /private/tmp/tmux-501/overmind-lorem-8gIald16MpjMK0sFPIg8Of (No such file or directory)
```

**Comment by DarthSim** [comment]
Does this happen for all your Procfiles?

**Comment by sdahlbac** [comment]
Sorry for the noise, I was being stupid, the actual thing being wrong was the following line missing from said Procfile

```
jwt_server: cd backend && docker-compose up jwt_server
```

i.e. I was trying to start a non-existent thing. 🤦  The error message was not perhaps the clearest in  pointing that out though.

**Comment by DarthSim** [comment]
I am the one who should be sorry, Overmind should definitely check this kind of stuff. Thanks for debugging! Fixed in v2.2.2
